tests/e2e/framework/istioctl.go

```diff
@@ -31,6 +34,7 @@ const (
 
 var (
 	remotePath = flag.String("istioctl_url", os.Getenv(istioctlURL), "URL to download istioctl")
+	pfPID      int
```

Comment:
I would move pfPID in the struct, and store the process directly instead of store the pid, that way you can kill directly the process like we saw with Dan.